.github/workflows/ci.yml

```diff
@@ -77,3 +77,50 @@ jobs:
             build/reports/tests/test
             build/reports/jacoco/test
         if: always()
+
+
+  docs:
+    name: 'Docs'
+    needs: test
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    env:
+      NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+      NETLIFY_SITE_ID: ${{ vars.NETLIFY_SITE_ID }}
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
```

Comment:
This job should use the `gradle-build-action` like the other ones, thus ensuring that the gradle cache is used.

Reply:
Nice catch... I was wondering if this setup step was still required when using the repo's wrapper. I've read the action documentation and now and it makes sense. Thanks.

Reply:
It's technically not required as the `setup-java` action _can_ do it as well, but it's disabled by default and not as powerful, as we cannot set an explicit gradle version like we do right now